src/main/java/io/cucumber/core/plugin/SerenityReporter.java

```diff
@@ -896,7 +895,7 @@ private boolean errorOrFailureRecordedForStep(String stepTitle, Throwable cause)
     }
 
     private boolean isAssumptionFailure(Throwable rootCause) {
-        return (AssumptionViolatedException.class.isAssignableFrom(rootCause.getClass()));
+        return ("org.junit.internal.AssumptionViolatedException".equals(rootCause.getClass().getCanonicalName()));
```

Comment:
These statements are not equivalent though - the class may also be derived from `org.junit.internal.AssumptionViolatedException`

Reply:
Yes, better use Class.forName with a try-catch block.